Accessories/uInverter/DESIGN CALCULATIONS.md

```diff
@@ -0,0 +1,45 @@
+﻿## uInverter Design Calculations
+
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 9V input voltage. 
```

Comment:
Why `9V input voltage`? I thought you were designing for a 12V bus?

Please explicitly state your design specification in a section at the top of this document. Design specification should include: 
- nominal dc link voltage
- rated phase current
- range of allowable switching frequency
- range of allowable power frequency